.github/workflows/tests.yml

```diff
@@ -0,0 +1,93 @@
+name: Tests
+
+# NOTE(mhayden): Restricting branches prevents jobs from being doubled since
+# a push to a pull request triggers two events.
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - master
+
+jobs:
+  pylint:
+    name: "pylint"
+    runs-on: ubuntu-latest
```

Comment:
Since we have a ubuntu release specific runner down below (`f27-build-from-ubuntu1804.json`) should we maybe also pin the container env to the same release (`ubuntu-18.04`)?